src/components/atoms/Ranges/LevelRange/LevelRange.tsx

```diff
@@ -10,17 +10,19 @@ import {
 } from './LevelRange.styled';
 import { data } from './data';
 import { IProps } from './LevelRange.model';
+import newId from '../../utils/newid';
```

Comment:
We have support for absolute paths in TS

```suggestion
import newId from 'utils/newid';
```